lib/types/date/index.js

```diff
@@ -126,6 +127,11 @@ internals.Date = class extends Any {
         return internals.isoDate.test(value);
     }
 
+    _isIsoDuration(value) {
```

Comment:
I didn't want to put the regex for ISO duration into the string index.js, but rather follow the existing pattern. I also don't want to reach into date's internals from the string validator. 

I could just move this regex into the string validator, but I feel like it belongs next to the date regex.

Reply:
Well, since it's never going to be involved in the creation of a date, I'm not so sure. It could even be a number for all I know. I don't usually use those, how are you going to consume that data?